colossalai/inference/tensor_parallel/modeling/__init__.py

```diff
@@ -1,5 +1,3 @@
-import _utils
```

Comment:
import issue: preferably to be fixed in https://github.com/hpcaitech/ColossalAI/pull/4830